spa_ui/self_service/gulp/config.js

```diff
@@ -382,6 +382,25 @@ module.exports = (function() {
     }
   };
 
+  var poDir = 'client/gettext/po/';
+
+  config.gettextExtract = {
+    inputs: ['client/**/*.js', 'client/**/*.html'],
+    potFile: 'template.pot',
```

Comment:
I'm guessing we probably want to change `template.pot` to something more sensible,
such as `manageiq-ssui.pot` maybe?